src/components/Calculator.js

```diff
@@ -1,129 +1,117 @@
-import React from 'react';
+import useState from 'react-hook-use-state';
 
 import '../App.css';
 
 import calculate from '../logic/calculate';
 
-class Calculator extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-    this.onClick = this.onClick.bind(this);
-  }
-
-  onClick = (buttonName) => {
-    this.setState((prevState) => calculate(prevState, buttonName));
-  };
-
-  render() {
-    function display(total, operation, next) {
-      if (next !== null && operation === null) {
-        return next;
-      }
-      if (operation !== null && next !== null) {
-        return next;
-      }
+function Calculator() {
```

Comment:
Make use of the arrow function as mentioned 🤦 , is there any issue with it, seems you missed it a second time.

example:
```
const Calculator = () => {
  .......
}
```